kafka/config.go

```diff
@@ -24,6 +24,7 @@ import (
 	"fmt"
 	"github.com/Shopify/sarama"
 	"go.uber.org/zap/zapcore"
+	"time"
```

Comment:
nit: reorg imports

Reply:
the go fmt insist to import "time" at the bottom part.

Reply:
go fmt formats alphabetically within a block but you need to give it a hint.

If you reorder it as:
```
fmt
time

"github.com/Shopify/sarama"
"go.uber.org/zap/zapcore"
``` 
Then run go fmt, it should format it correctly.